src/test/resources/inputs/basics/paths.sql

```diff
@@ -157,12 +157,12 @@ SELECT "t"."flds"."a"."field_x" AS v FROM EXCLUDE_T AS t;
 
 -- nullable fields
 --#[paths-sfw-21]
-SELECT t.flds.a.field_x AS v FROM EXCLUDE_T_NULLABLE AS t;
+SELECT t.flds.a.field_x AS v FROM EXCLUDE_T AS t;
 
 -- nullable fields + qualified path
 --#[paths-sfw-22]
-SELECT "t"."flds"."a"."field_x" AS v FROM EXCLUDE_T_NULLABLE AS t;
+SELECT "t"."flds"."a"."field_x" AS v FROM EXCLUDE_T AS t;
 
 -- nullable fields + mix of qualified and unqualified path components
 --#[paths-sfw-23]
-SELECT "t".flds."a".field_x AS v FROM EXCLUDE_T_NULLABLE AS t;
+SELECT "t".flds."a".field_x AS v FROM EXCLUDE_T AS t;
```

Comment:
self-review: change to not use the `EXCLUDE_T_NULLABLE` tables since they have been deleted